bundles/org.openhab.binding.ring/src/main/java/org/openhab/binding/ring/internal/RingVideoServlet.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2010-2025 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+
+package org.openhab.binding.ring.internal;
+
+import static org.openhab.binding.ring.RingBindingConstants.*;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.OutputStream;
+import java.net.URLConnection;
+import java.nio.file.FileSystem;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.eclipse.jetty.http.HttpMethod;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Main OSGi service and HTTP servlet for Ring Video
+ *
+ * @author Peter Mietlowski (zolakk) - Initial contribution
+ * @author Ben Rosenblum - Updated for OH4 / New Maintainer
+ */
+@Component(service = HttpServlet.class)
+@NonNullByDefault
+public class RingVideoServlet extends HttpServlet {
+
+    private static final long serialVersionUID = -5592161948589682812L;
+
+    private final Logger logger = LoggerFactory.getLogger(RingVideoServlet.class);
+
+    private String videoStoragePath = "";
+    /**
+     * The RestClient is used to connect to the Ring Account.
+     */
+    private @Nullable RestClient restClient;
```

Comment:
Why do you need that? It's not used in the code

Reply:
Correct.